### ISSUE TITLE
[Terminology] Replace occurrences of "master" with "primary"

### ISSUE THREAD

**Issue by perry-mitchell** [buttercup/buttercup-core#230]
Replace occurrences of "master" with "primary" or something else more appropriate. Datasources/Credentials will be affected.

**Comment by wolffles** [comment]
Whats wrong with calling it master? Will the replacements be throughout all of buttercup-core?

**Comment by perry-mitchell** [comment]
@wolffles Yeah this was never intended to be a high priority change, but a potentially necessary one as the terminology (at least in tech) is in somewhat international contention at the moment. See the [python master/slave terminology update](https://motherboard.vice.com/en_us/article/8x7akv/masterslave-terminology-was-removed-from-python-programming-language).

We may not adopt this at all, so I changed the status to pending. Imho Master/Slave conveys no negative connotations when used in tech but there are for sure others that have different opinions and may be offended or put-off by their use in code/UIs. We use "Master Password" quite prominently in our UI, and although we don't use the term _slave_ to my knowledge, the two go somewhat hand in hand.

This was just intended to avoid _potential_ issues with terminology as there are perfectly fine alternatives. Let's see how we go..
